packages/bpk-component-modal/README.md

```diff
@@ -85,7 +85,7 @@ class App extends Component {
 | isOpen                | bool     | true     | -             |
 | onClose               | func     | true     | -             |
 | getApplicationElement | func     | true     | -             |
-| dialogRef             | func     | true     | -             |
+| dialogRef             | func     | false    | () => null    |
```

Comment:
this is not actually required. In fact, I believe passing it doesn't do anything so would be good to just remove this later on (didn't want to make this a breaking change right now)